### DISCUSSION TITLE
TIMESTAMP to date time format

### DISCUSSION THREAD

**Discussion by rohitgarud** [holdenmatt/duckdb-wasm-kit#9] [General]
Hi.. can you please add the Discussion section to this repo, so that we can ask general questions there.

My question is that I am importing from a csv file but the date columns are of TIMESTAMP type with time given in microseconds from the epoch, I want to have these columns in ISO datetime format.. how can I convert columns from the uploaded to the required format?

**Comment by holdenmatt** [comment]
This library isn't doing type conversion like that, it uses DuckDB's default CSV parsing behavior, so an int column will probably be imported as int. You should be able to cast the column to a different type using DuckDB's date-related functions.

I haven't verified if this works, but try this: https://chat.openai.com/share/093e50a6-8ab0-4c2f-8992-2ca71e8a00d2

**Reply by rohitgarud** [reply]
Thank you for your response.. I have gone through Duckdb-wasm documentation as well.. was not sure how exactly to implement it..

**Reply by rohitgarud** [reply]
Adding time_format argument to insertCSV, insertParquet functions of this package might be a good feature to have
